lib/generators/manageiq/provider/provider_generator.rb

```diff
@@ -101,7 +101,6 @@ def create_scaffolding
       template "app/models/%plugin_path%/inventory/persister.rb"
       template "app/models/%plugin_path%/inventory/collector/%manager_path%.rb"
       template "app/models/%plugin_path%/inventory/parser/%manager_path%.rb"
-      template "app/models/%plugin_path%/inventory/persister/definitions/cloud_collections.rb"
```

Comment:
Unrelated change?

Reply:
Ah yeah I thought I made a separate PR for this